src/Estimators/NEReferencePoints.h

```diff
@@ -0,0 +1,67 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2022 QMCPACK developers.
+//
+// File developed by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//                    Mark A. Berrill, berrillma@ornl.gov, Oak Ridge National Laboratory
+//                    Peter W. Doak. doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// File refactored from: QMCHamiltonian/ReferencePoints.h
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_NEREFERENCE_POINTS_H
+#define QMCPLUSPLUS_NEREFERENCE_POINTS_H
+
+#include <Configuration.h>
+#include "OhmmsData/OhmmsElementBase.h"
+#include "Particle/ParticleSet.h"
+#include "QMCHamiltonians/ObservableHelper.h"
+#include "OhmmsPETE/Tensor.h"
+#include "ReferencePointsInput.h"
+
+namespace qmcplusplus
+{
+class NEReferencePoints
```

Comment:
I have a hard time to figure out what this class does and what it is for. Need documentation.

Reply:
Yes but I'll give it a little more doc.